plugins/direction-reveal.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "146",
+  "title": "Direction Reveal: Direction aware content reveals",
+  "image": "/direction-reveal.jpg",
```

Comment:
this need to match the file's name, so it should be `146-direction-reveal.jpg` instead of `direction-reveal.jpg`

Reply:
Sorry for the mix up. I have updated the file name.